### ISSUE TITLE
Level: Rabucket

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#539]
For #520 - now with a solution, and  a possible candidate for a hard level?

```
 A  Q  A 
 #     # 
 y     t 
   / \   
  p #    
#O# # #O#
### # ###
    #    
:author_name=Andy Balaam
:author_url=http://artificialworlds.net/rabbit-escape
:bash=2
:bridge=2
:brolly=1
:description=Empty the buckets!
:dig=1
:name=Rabucket
:num_rabbits=2
:num_to_save=2
:rabbit_delay=15
:solution.1.code=gnoI"tnR+Oy ,:u,s+goOoyStoNNNoS-ino,itnl-uPg,64+b-ooRopNOOStOgNgigounyDonsooNSO0R9E
```

**Comment by GamingInky** [comment]
New solution in PR #544

**Comment by andybalaam** [comment]
Thanks for finding this solution!

This solution makes me sad.  I have merged it, but if anyone can think of how to prevent it, we can change it to a non-solution (until:LOST) and then I would be happy again.

**Comment by tttppp** [comment]
I got:
```
D;k'\=&mFW~F39'&"~<r9\B$u~
```
Which can be decoded with some escaping - e.g. 
```
runrabbit --decode $'D;k\'\=&mFW~F39\'&"~<r9\B$u~'
```

This seems slightly more trivial than the existing solutions.

@GamingInky's solution can be prevented by moving the entrance up one square (as then the second rabbit will plunge to their death).  Mine could be prevented with more fire or bomb tokens (although this will make it a little more obvious what the intended solution is).

I think this is currently on a par with Red Army and Double Showdown (i.e. it should go somewhere around level 4 or 5).

**Comment by andybalaam** [comment]
Maybe we just accept its current difficult level.  We could try moving the exits further from the centre to prevent tttppp's solution, but the rabbots being able to bash is definitely a pity, and I don't see how to prevent it.  I guess the answer would be to dig instead of bashing, somehow.

**Comment by andybalaam** [comment]
Released in 0.11

**Comment by colonelfazackerley** [comment]
I found @tttppp's solution independently, on the first attempt.

`./runrabbit --decode $'D;k\'\=&mFW~F39\'&"~<r9\B$u~'`

this one is a bit easy for it's position in the set.
